src/Tickets/Commerce/Gateways/PayPal/Client.php

```diff
@@ -131,19 +137,52 @@ public function get( $endpoint, array $query_args = [], array $request_arguments
 				'Content-Type'  => 'application/json',
 			]
 		];
+
+		if ( 'GET' !== $method ) {
+			$default_arguments['body'] = [];
+		}
+
 		foreach ( $default_arguments as $key => $default_argument ) {
 			$request_arguments[ $key ] = array_merge( $default_argument, Arr::get( $request_arguments, $key, [] ) );
 		}
-		$response = wp_remote_get( $url, $request_arguments );
+
+		if ( 'GET' !== $method ) {
+			$content_type = Arr::get( $request_arguments, [ 'headers', 'Content-Type' ] );
+			if ( empty( $content_type ) ) {
+				$content_type = Arr::get( $request_arguments, [ 'headers', 'content-type' ] );
+			}
+
+			if (
+				! empty( $request_arguments['body'] )
+				&& 'application/json' === strtolower( $content_type )
+			) {
+				$request_arguments['body'] = wp_json_encode( $request_arguments[ $key ] );
+			}
+		}
+
+		if ( 'GET' === $method ) {
```

Comment:
```suggestion
		} elseif ( 'GET' === $method ) {
```

Reply:
I think this might be intentional, otherwise the `elseif` with the `'POST'` would never run

Reply:
Exatcly.